src/tls/server.c

```diff
@@ -645,8 +640,11 @@ server_poll_event (int timeout)
     {
       if (ev.data.ptr == &server)
         handle_accept ();
+      else if (ev.data.ptr == handle_child_exit)
+        handle_child_exit ();
       else if (ev.events & EPOLLIN)
         handle_connection_data (ev.data.ptr);
+
```

Comment:
The smallest nitpick of course, but it seems nicer to keep the else-if ladder together. Adding an empty line here makes it look like the next block was independent.